addons/plotting/CfgFormatters.hpp

```diff
@@ -0,0 +1,30 @@
+class GVAR(formatters) {
+    class Distance {
+        class Meter {
+            formatter = QUOTE(format [ARR_2('%1 m',_value toFixed 1)]);
+            priority = 100;
+        };
+        class Feet {
+            formatter = QUOTE(format [ARR_2('%1 ft',(_value * 3.281) toFixed 1)]);
+            priority = 90;
+        };
+        class Yards {
+            formatter = QUOTE(format [ARR_2('%1 yd',(_value * 1.094) toFixed 1)]);
+            priority = 80;
+        };
+        class Mile {
+            formatter = QUOTE(format [ARR_2('%1 mi',(_value / 1609.344) toFixed 2)]);
+            priority = 70;
+        };
```

Comment:
I think these could be made more complex. For example, I don't think the decimal point is needed after 100 m and for any moving object it just flickers. Essentially, increase precision for small values, decrease for large ones.

Reply:
Good idea